public/app/core/utils/types.ts

```diff
@@ -0,0 +1,3 @@
+type Truthy<T> = T extends false | '' | 0 | null | undefined ? never : T;
```

Comment:
Should we add it to grafana-data/types?

Reply:
it is not exported by grafana-data, i just moved it from the ES datasource to somewhere more sensible to be reused, i don't think this and `isTruthy` belongs to @grafana/data